packages/tools/devtools/devtools-core/src/ContainerDevtools.ts

```diff
@@ -288,7 +291,11 @@ export class ContainerDevtools implements IContainerDevtools, HasContainerKey {
 		[DisconnectContainer.MessageType]: async (untypedMessage) => {
```

Comment:
Note that the change to behavior here only has impact if we disconnect the Container via the UI in the devtools. If the Container disconnects for any other reason, we'll continue to have the original issue. So I don't think this is the right place to apply a fix.